python/cugraph-pyg/cugraph_pyg/tests/nn/test_gat_conv.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+
+try:
+    from torch_geometric.nn import GATConv
+except ModuleNotFoundError:
+    pytest.skip("PyG not available", allow_module_level=True)
+
+from cugraph.utilities.utils import import_optional
+from cugraph_pyg.nn import GATConv as CuGraphGATConv
+
+torch = import_optional("torch")
+
+
+@pytest.mark.parametrize("bias", [True, False])
+@pytest.mark.parametrize("bipartite", [True, False])
+@pytest.mark.parametrize("concat", [True, False])
+@pytest.mark.parametrize("heads", [1, 2, 3])
```

Comment:
1,2,3 is not too great as they end up calling all into the same kernel, eventually use something out of [1, ..., 4] and [5, ..., 16]